types/plugin/duration.d.ts

```diff
@@ -77,3 +86,5 @@ declare module 'dayjs' {
   export const duration: plugin.CreateDurationType;
   export function isDuration(d: any): d is plugin.Duration
 }
+
+type a = FnPa<plugin.AddDurationType>;
```

Comment:
what's this line used for, please?

Reply:
I wanted to add this line to @deprecated as a recommendation. I'm not sure how it was included into the last commit, but I have remove it. Check it out